### ISSUE TITLE
What to do with the wiki

### ISSUE THREAD

**Issue by sypets** [TYPO3-Documentation/T3DocTeam#8]
# Problem

The wiki contains a number of outdated pages. 

# Additional consideration

* the wiki in general can't be removed unless the exception messages are moved elsewhere

# Possible solutions

* remove outdated content (and add redirects)
* Sift through the remains and see what is still useful
* Move relevant things from Wiki to docs.typo3.org, 

The wiki is also used for team communication (e.g. t3board). This must be considered seperately and it beyond of scope of this issue.

# Tasks

* [x]  make assessment of what is in the wiki: https://wiki.typo3.org/OverviewOfWiki
* [ ]  Decide on a workflow: see #99 (see https://wiki.typo3.org/WikiMigration for first draft)
* [ ]  Remove outdated pages and create redirects (in progress)
* [ ] Move content that is still relevant, merge with docs.typo3.org and create redirrects

# Open decisions

* [ ] what to do with existing documentation in other languages (in the Wiki)
* [ ] what to do about team communication (e.g. t3board)
* [ ] what to do with exceptions
* [ ] What to do with extension documentation (that does not already live elsewhere)? See also [Slack](https://typo3.slack.com/archives/C028JEPJL/p1566735573410700)

# Additional information

* see overview in Wiki: https://wiki.typo3.org/OverviewOfWiki

**Comment by DanielSiepmann** [comment]
One suggestion of @randomresult was to shut down the wiki. We should export everything as read only static pages. This way no new stuff can make it into the wiki and old stuff, which is still relevant, can be migrated. Once everything is done, we should remove everything.

**Comment by sypets** [comment]
IMO **before** we create static pages, we should do spring cleaning and remove outdated / irrelevant pages. Otherwise the static pages are difficult to edit and will live forever and clutter up the Google results.

**Comment by sypets** [comment]
I looked at some of the pages in the wiki. It is *a lot*. I am more and more in favor of removing it all and moving the exception messages to docs.typo3.org. 

What someone should do though, is provide and point people to alternatives (but that is not the job of the docs team), e.g. for team communication. See also updated descriptions for list of what Wiki is currently being used for.

**Comment by sypets** [comment]
I think an assessment of what is currently in the Wiki is better maintained in the Wiki itself, so I started a page there: https://wiki.typo3.org/OverviewOfWiki

Also, I think this issue cannot be handled just by the docs team itself. The Wiki does not only contain docs, it contains exception messages (core team), sprint planning (core team) and team and event communication and other stuff.

**Comment by DanielSiepmann** [comment]
@marble will adjust the robots.txt to prevent further indexing of the wiki.

**Comment by sypets** [comment]
During team sprint in 2018, the documentation team decided, that it is not responsible for Wiki, see [# 26 comment](https://github.com/TYPO3-Documentation/T3DocTeam/issues/26#issuecomment-408606287).

**Comment by sypets** [comment]
After some more reading documentation and looking through the Wiki, I would strongly advise to give it some more thought and look at existing content before shutting it down or making it read-only.

Because:
* some of the pages in the Wiki are still useful and being linked to. There is currently no alternative. We have been merging a lot lately, but some of the pages can't just be easily merged, because the Wiki is structured in a page by page basis, but docs.typo3.org is structured by manuals or main sections. Before we move individual pages we should think about the structure, e.g. should it just be a long list of individual pages or what? 
* Some things in the Wiki are actually quite useful, for example the Categories, the Discussions, there are categories like "Needs Review" etc. that could be used for a proper review workflow (something that is currently missing in the official docs).
* Editing the Wiki is actually easier than editing the docs: you have a working preview, do not have to render locally, and the syntax has less gotchas and traps you can fall into

The main thing wrong with the Wiki is that is not being updated, maintained enough. But why should that change if / when the pages are moved to docs.typo3.org? If some pages are hopelessly outdated, we can delete them, but that is not a reason to delete entire wiki. (**Update**: Got a good answer here: Anyone can edit Wiki, but changes to docs need to be merged, so you can enforce quality. However: hasn't always worked in the past, see lots of spelling mistakes, errors and outdated docs).

Another thing is the layout is not as pretty and modern as it might be. That is also something that someone could work on. 

What I would advise to do though is: Do not mix documentation and team communication in Wiki. Have separate platforms for these things.

**Comment by sypets** [comment]
Created project for Wiki isues and added this issue: https://github.com/orgs/TYPO3-Documentation/projects/3

**Comment by linawolf** [comment]
I sorted to the whole category "Extensions". found 13 manuals that are exclusively in the Wiki and where downloaded by at least 500ppl. Put all of them in a category: https://wiki.typo3.org/Category:ExtensionsOnlyDocumentation I made issues in the Git repositories of cooluri and realurl to move documentation.
What should we do about the remaining manuals?
What should we do about non-english tutorials and manuals? I didnt check them
